apps/condo/domains/user/schema/ResetUserService.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Generated by `createservice user.ResetUserService --type mutations`
+ */
+
+/**
+ * Reset user mutation removes phone, email and password from user, making it impossible to use this user.
+ *
+ * Mutation also sets name to "Deleted User" and do not set deletedAt.
+ * Reason for this is the fact that if we soft-delete user, then we are about to face a lot of "user is null" kind of errors in unpredictable places.
+ * Name is changed so we can distinguish reset users.
+ *
+ * Telegram uses the same approach: if user is deleted, then all his messages and other content are available, but his name is now "Deleted user"
+ *
+ * Primary use cases:
+ * 1. QA Wants to test SMS services, but dont want to get multiple phone numbers
+ * 2. User wants to remove all his data from our system.
+ */
+
+const { GQLCustomSchema, getById } = require('@core/keystone/schema')
+
+const access = require('@condo/domains/user/access/ResetUserService')
+const { DELETED_USER_NAME } = require('@condo/domains/user/constants')
+const { User } = require('@condo/domains/user/utils/serverSchema')
+
+const { DV_UNKNOWN_VERSION_ERROR, NOT_FOUND_ERROR } = require('@condo/domains/common/constants/errors')
+
+
+const ResetUserService = new GQLCustomSchema('ResetUserService', {
+    types: [
+        {
+            access: true,
+            type: 'input ResetUserInput { dv: Int!, sender: JSON! id: String!}',
```

Comment:
```suggestion
            type: 'input ResetUserInput { dv: Int!, sender: SenderInput!, id: String!}',
```

Reply:
It’s better to Use user link type instead of just Id

Reply:
we discussed it in calll